infra/bridge/src/controllers/phone.js

```diff
@@ -130,7 +131,15 @@ router.post('/verify', phoneVerifyCode, async (req, res) => {
       verified: true
     }
   }
-  const attestationValue = `${req.body.country_calling_code} ${req.body.phone}`
+  let attestationValue
+
+  try {
+    attestationValue = parsePhoneNumberFromString(
+      `+${req.body.country_calling_code} ${req.body.phone}`
+    ).number
+  } catch {
+    attestationValue = `${req.body.country_calling_code} ${req.body.phone}`
```

Comment:
nit: log an error here so that we notice if the parsing fails.

Reply:
Done.